src/test/java/io/lettuce/core/cluster/pubsub/RedisClusterPubSubConnectionIntegrationTests.java

```diff
@@ -37,7 +41,7 @@
  * @author Mark Paluch
  */
 @ExtendWith(LettuceExtension.class)
-class RedisClusterPubSubConnectionIntegrationTests extends TestSupport {
+class RedisClusterPubSubConnectionIntegrationTests extends TestSupport implements RedisClusterPubSubListener<String, String> {
```

Comment:
I would suggest to not use the test as listener and instead, let a inner class implement  `RedisClusterPubSubListener` for a cleaner design.

Reply:
done